designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/theme/Ripple.kt

```diff
@@ -21,4 +21,20 @@ object Ripple {
         @Composable
         override fun rippleAlpha(): RippleAlpha = alpha
     }
+
+    internal object CustomDHISErrorRippleTheme : RippleTheme {
+
+        private val alpha = RippleAlpha(
+            focusedAlpha = 0.16f,
+            draggedAlpha = 0.16f,
+            hoveredAlpha = 0.04f,
+            pressedAlpha = 0.16f,
+        )
+
+        @Composable
+        override fun defaultColor(): Color = SurfaceColor.Error
+
+        @Composable
+        override fun rippleAlpha(): RippleAlpha = alpha
+    }
```

Comment:
If the only thing that's changing from the default ripple theme is color. Should we just add it as a param to the `CustomDHISRippleTheme` rather than creating a new one?